packages/scss/src/components/_label.scss

```
@@ -30,3 +30,14 @@
 		}
 	}
 }
+
+.label {
+	&.mod-number {
+		border-radius: _theme("sizes.standard.line-height");
+		padding: _theme("spacings.reset");
```

Actually you should have 
```suggestion
		padding: 0 _theme("spacings.smaller");
```
It would play better with larger numbers